### PR TITLE
Rewrite the video demo

### PR DESCRIPTION
**Pull request by fnevgeny**
I'm not certain whether we are allowed to distribute the Apollo movie. Perhaps should use a URL link instead of the file. This would also eliminate the need for the zip packaging and greatly reduce the size - a single PDF would be enough.

### CODE REVIEW

**Comment by andybarry** [comment]
The Apollo movie is in the public domain.  That's why I chose it :)

https://commons.m.wikimedia.org/wiki/File:Ap17-ascent.ogv

When I made the demo, I wanted it to be totally self-contained to reduce the number of confounding factors to deal with when figuring out why a system didn't work, so I think that distributing the movie is worthwhile.

But, I've been gone for years, so all good whatever you guys want to do.

File: demo/pdfpc-video-example/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Config
+DEMO  = video-example
+
+PDFDOCS = $(DEMO).pdf
+
+# Targets
+all : $(PDFDOCS)
+
+clean :
+	$(RM) $(PDFDOCS) *.aux *.log *.toc *.nav *.out *.snm *.vrb
```
Comment (AndreasBilke) [inline demo/pdfpc-video-example/Makefile:22]: $(RM) variable is not defined. So a `make clean` raises an error for me.

**Review by AndreasBilke — CHANGES_REQUESTED** [review]
The missing $(RM) is also in `demo/Makefile`.

I would propose in general to stop shipping `Makefiles` for latex code (they are always a mess) and use `latexmk` on your own instead.

If you prefer the `Makefile` then `make clean` should work :-)

File: demo/pdfpc-video-example/Makefile (hunk 1 above)
Comment (fnevgeny) [reply demo/pdfpc-video-example/Makefile:22]: <https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html>

Hmm, I didn't realize it was GNU-specific...

**Pull request merged**